### DISCUSSION TITLE
Problems with uploading proof of academic status

### DISCUSSION THREAD

**Discussion by jtan230** [community/community#122104] [GitHub Education]
### Hi everyone,

Github states that "you can upload a letter on school letterhead or any documentation with a date that demonstrates your current enrollment". However, as I am a graduate student, the letter of certification my school provides does not give a date for my graduation, only my estimated duration of candidature. I've tried uploading this document but I got prompted to upload another document. My student ID also does not have a date to verify my enrollment. May I request for a manual confirmation of my student status?

**Comment by davevad93** [comment]
Hi @jtan230 ,

A similar thing happened to me last year when I had to submit a proof of my academic status to obtain the student benefits. 
I've uploaded my ID and my enrollment sheet but neither had a date, so GitHub asked me for another proof, then I've uploaded a screenshot of my last exam results which showed a date and finally my request was accepted. Try doing the same as I did.

**Comment by jtan230** [comment]
Hi @davevad93, thanks so much for replying! I'll try doing what you've suggested after I receive my exam results, thanks again!

**Reply by davevad93** [reply]
You're welcome @jtan230 , I hope this solution may be useful to you. Have a nice day!
